pkg/cloudprovider/aws/suite_test.go

```diff
@@ -418,6 +418,8 @@ var _ = Describe("Allocation", func() {
 				Expect(createFleetInput.LaunchTemplateConfigs).To(HaveLen(1))
 				launchTemplate := createFleetInput.LaunchTemplateConfigs[0].LaunchTemplateSpecification
 				Expect(*launchTemplate.LaunchTemplateName).To(Equal(*createLaunchTemplateInput.LaunchTemplateName))
+				//TODO; make this optional/configurable
```

Comment:
Make sure to remove before submitting. We track these in issues, rather than TODO statements.